releases/2.12.16.md

```diff
@@ -0,0 +1,41 @@
+This release improves compatibility with recent JDKs:
+
+* For JDK 19 support, use ASM 9.3 ([#10000](https://github.com/scala/scala/pull/10000))
+* Make `-target` support JDK 8 through 19 (and deprecate 5 through 7) ([#9916](https://github.com/scala/scala/pull/9916))
+* Fix codegen for `MethodHandle.invoke` (et al) under JDK 17 `-release` ([#9930](https://github.com/scala/scala/pull/9930))
+* Deprecate `AnyVal#formatted(formatString)`, to avoid conflict with JDK 15+ method ([#9783](https://github.com/scala/scala/pull/9783))
+
+and aids cross-compiling with Scala 3:
+
+* Allow `?` as a wildcard even without `-Xsource:3` ([#10005](https://github.com/scala/scala/pull/10005))
+
+and improves security and error reporting:
+
+* Error on source files with Unicode directional formatting characters ([#10023](https://github.com/scala/scala/pull/10023), [#10030](https://github.com/scala/scala/pull/10030))
+* Fix range position end for infix calls ([#9761](https://github.com/scala/scala/pull/9761))
+
+For complete 2.12.16 change lists, see [all merged PRs](https://github.com/scala/scala/pulls?q=is%3Amerged%20milestone%3A2.12.16) and [all closed bugs](https://github.com/scala/bug/issues?utf8=%E2%9C%93&q=is%3Aclosed+milestone%3A2.12.16).
+
+## Compatibility
+
+As usual for our minor releases, Scala 2.12.16 is binary-compatible with the whole Scala 2.12 series.
+
+## Contributors
+
+A big thank you to everyone who's helped improve Scala by reporting bugs, improving our documentation, spreading kindness in discussions around Scala, and submitting and reviewing pull requests! You are all magnificent.
+
+This release was brought to you by 18 contributors, according to `git shortlog -sn --no-merges @ ^v2.12.15 ^2.11.x`. Thank you Scala Steward, Seth Tisue, Lukas Rytz, Sébastien Doeraene, A. P. Marki,Daniel Le, Jason Zaugg, Guillaume Martres, Frank Thomas, Philippus, Daniela Sfregola, Dale Wijnand, Vadzim Marchanka, hepin.p, naferx, Fredrik Svensson, Štefan Jurčo, Georgi Krastev.
+
```

Comment:
Suggest a space before Daniel Le. A space in backticks looks like [` `]. :confused:

Reply:
done

Reply:
I forgot to make a "space invaders" joke. I just thought of it because someone is playing Blondie, "Call me." I wonder if that is an old joke from the "tabs vs spaces" wars of the '80s-'90s.